src/einsteinpy/utils/schwarzschild_utils.py

```diff
@@ -1,11 +1,10 @@
 import astropy.units as u
 import numpy as np
 
-from einsteinpy import constant
+from einsteinpy import constant, utils
 
 
-@u.quantity_input(mass=u.kg)
-def schwarzschild_radius(mass):
+def schwarzschild_radius(mass, c=constant.c.value, G=constant.G.value):
```

Comment:
I was saying if you could change `c=constant.c.value` to `c=constant.c` and `G=constant.G.value` to `constant.G`

Reply:
still getting error